mysql-test/suite/mariabackup/MDEV-31042_3.test

```diff
@@ -0,0 +1,48 @@
+# This is to make sure that the xtrabackup_tmp_#<tablespace_id> is removed when not utilized.
+
+--source include/have_file_key_management.inc
+--source include/have_debug.inc
```

Comment:
Tests should not be skipped on non-debug builds unless there is a very good reason. I do not see any fault injection being present in this test.